### ISSUE TITLE
[BUG] Nice Repo! But when use CUP algorithm's configs doesn't have lagrangian_upper_bound

### ISSUE THREAD

**Issue by yinglinmonian** [PKU-MARL/omnisafe#45]
### Required prerequisites

- [X] I have read the documentation <https://omnisafe.readthedocs.io>.
- [X] I have searched the [Issue Tracker](https://github.com/PKU-MARL/omnisafe/issues) and [Discussions](https://github.com/PKU-MARL/omnisafe/discussions) that this hasn't already been reported. (+1 or comment there if it has.)
- [X] Consider asking first in a [Discussion](https://github.com/PKU-MARL/omnisafe/discussions/new).

### What version of OmniSafe are you using?

0.0.2

### System information

0.0.2

### Problem description

In this file https://github.com/PKU-MARL/omnisafe/blob/dev/omnisafe/algorithms/on_policy/cup.py#L57.
The codes are, 
```python
        Lagrange.__init__(
            self,
            cost_limit=self.cfgs.lagrange_cfgs.cost_limit,
            lagrangian_multiplier_init=self.cfgs.lagrange_cfgs.lagrangian_multiplier_init,
            lambda_lr=self.cfgs.lagrange_cfgs.lambda_lr,
            lambda_optimizer=self.cfgs.lagrange_cfgs.lambda_optimizer,
            lagrangian_upper_bound=self.cfgs.lagrange_cfgs.lagrangian_upper_bound,
        )
```
But the CUP's configs yaml files doesn't have lagrangian_upper_bound in lagrange_cfgs.

### Reproducible example code

The Python snippets:

```python

```

Command lines:

```bash

```

Extra dependencies:

```text

```

Steps to reproduce:

1.
2.
3.

**Comment by zmsn-2077** [comment]
Sorry, we have solved the problem in PR #44  by simply removing the `lagrangian_upper_bound` pass in the `CUP` and using the default None.
<img width="878" alt="image" src="https://user-images.githubusercontent.com/73586554/208264828-6f5519ba-0f42-4986-bc51-279af964b199.png">
I hope this can helps you.

**Comment by yinglinmonian** [comment]
Thanks.
